lib/bundix.rb

```diff
@@ -143,7 +143,7 @@ def build_depcache(lock)
             dep_cache[name] = Dependency.new(name, lock.bundler_version, {})
           end
 
-          if !((as_dep.groups - cached.groups) - [:default]).empty? or !(as_dep.platforms - cached.platforms).empty?
+          if !(as_dep.groups - cached.groups).empty? or !(as_dep.platforms - cached.platforms).empty?
```

Comment:
I believe `[:default]` here is just a workaround to the wrong group set for transitive dependencies. This is also what breaks https://github.com/nix-community/bundix/issues/39